.gitlab/gitlab-ci-crusher.yml

```diff
@@ -0,0 +1,223 @@
+# Ad-hoc build that runs in the ECP Hardware, concretely in OLCF Crusher.
+
+stages:
+  - pre
+  - setup
+  - build
+  - post
+
+.setup_env_ecpci: &setup_env_ecpci |
+  git fetch
+  source scripts/ci/gitlab-ci/setup-vars.sh
+  git checkout "$CI_ORIGINAL_SHA"
```

Comment:
Am I just misreading how things are working here?  It looks like you are still checking out the PR HEAD commit to test, rather than the merge commit made by github (which is what the sync script pushes).

Reply:
I made changes in `setup-vars.sh` resulting in CI_ORIGINAL_SHA to be UPSTREAM_COMMIT_SHA if UPSTREAM_COMMIT_SHA is defined

Reply:
Weird I'm not seeing that.

Reply:
> I should git checkout "$GIT_COMMIT_REF"

Ok, that makes sense.

Reply:
resolved